purview/models/__init__.py

```diff
@@ -81,3 +37,62 @@ def replace(self, node, properties=None):   # pragma: no cover
 
 RelationshipManager.disconnect_all = disconnect_all
 RelationshipManager.replace = replace
+
+
+def story_flow(label):
```

Comment:
The variable name `label` is misleading because I assume it to mean it's a Neo4j label which is now no longer true (e.g. `ContainerKojiBuild`.)